product/carbon-home/conf/identity/profile-mapping.yaml

```diff
@@ -149,3 +149,21 @@ profiles:
    readonly: false
    regex: "*"
    dataType: "email"
+ -
+  profileName: "user-identifier"
```

Comment:
What is the reason to add another user profile? Since you have marked it as a non admin profile, this profile will be shown in the account-settings page of users. Is that your intention?